include/simdjson/document.h

```diff
@@ -389,25 +389,29 @@ class element : protected internal::tape_ref {
   really_inline bool is_null() const noexcept;
 
   /**
-   * Tell whether the value can be cast to the given primitive type.
+   * Tell whether the value can be cast to provided type (T).
    *
    * Supported types:
    * - Boolean: bool
    * - Number: double, uint64_t, int64_t
    * - String: std::string_view, const char *
-   * - Array: array
+   * - Array: dom::array
+   * - Object: dom::array
```

Comment:
dom::object (same below).

Reply:
Thanks for catching this typo. Will fix.

Reply:
Fixed.